### DISCUSSION TITLE
Compatibility with selectize.js

### DISCUSSION THREAD

**Discussion by bytestream** [orchidjs/tom-select#77] [Q&A] [answered]
Hi @oyejorge 

Apologises if this has been answered already but it wasn't immediately clear to me after checking the website and README.  I'm wondering about compatibility of this fork compared to selectize.js, in other words, how difficult it is to switch from selectize.js to your fork.

Other than the divergence in initialisation:
```js
new TomSelect('#tom-select-it',config);
```

Is `config` fully compatible with selectize.js? If so, then I would plan to create a thin jQuery wrapper so that all existing usages of `$().selectize()` work without changes.

**Comment by oyejorge** [comment, marked as answer]
There are few things that selectize users might need to do when switching over. There's a list of breaking changes on https://tom-select.js.org/docs/selectize.js/ which is mainly the following:

- .ts-* css class names instead of .selectize-* (customizable with scss & js)
- scss instead of less
- dataAttr defaults to null instead of "data-data"
- options must be appended to optgroup within custom optgroup template

> Is config fully compatible with selectize.js?

Except for the template.optgroup and the dataAttr changes, a selectize config should work fine with Tom Select.

>  If so, then I would plan to create a thin jQuery wrapper so that all existing usages of $().selectize() work without changes.

Sounds like a great idea. Let me know if there's anything I can do
